lib/cpp/src/thrift/protocol/THeaderProtocol.cpp

```diff
@@ -146,6 +146,10 @@ uint32_t THeaderProtocol::writeBinary(const std::string& str) {
   return proto_->writeBinary(str);
 }
 
+uint32_t THeaderProtocol::writeUUID(const TUuid& uuid) {
```

Comment:
@Jens-G This is a bugfix for the THeaderProtocol.
Can I just create a Jira Ticket for it and we close it part of this PR (once I get there) or do I need to pull this out into a new PR for the issue?